MVVMRxSwiftDemo/Sources/Presentation/Modules/Home/Cell/SurveyCollectionCell.swift

```diff
@@ -34,16 +34,16 @@ final class SurveyCollectionCell: UICollectionViewCell {
     }
 
     func bind(_ viewModel: SurveyCollectionCellViewModel) {
```

Comment:
A little update here, we can use `SurveyCollectionCellViewModelProtocol` 
```suggestion
    func bind(_ viewModel: SurveyCollectionCellViewModelProtocol) {
```